vng_api_common/descriptors.py

```diff
@@ -34,7 +35,7 @@ def __init__(
         ), "The fields in 'optional' must be a subset of the mapping keys"
 
         # check if it's optional or not
-        self.required = any(field.blank is False for field in self.mapping.values())
+        self.required = required
```

Comment:
This seems a little strong. Can you set the default for `required` to `None`, and keep the old behaviour if `required is None`?